lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -118,6 +126,9 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_TIME, M
    */
   this.notify = () => {
     return this.getLatestVersion().then((version) => {
+      AnalyticsService.logEvent('Notifying update', {
```

Comment:
All the update related events are handled at `app.js`. It feels weird that this one is here. Let's either move all update events to this module, or move this to `app.js`.